contracts/internals/controller.sol

```diff
@@ -25,6 +25,7 @@ import "./transferrable.sol";
 interface IController {
     function isController(address) external view returns (bool);
     function isAdmin(address) external view returns (bool);
+    function isStopped() external view returns (bool);
```

Comment:
think all of this can be 'hidden' in the Controller ... as in isController() can have `require(!_stopped, "should not be stopped")`

Reply:
yes, doing this now!